h/admin/views/cohorts.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+from pyramid.view import view_config
+from pyramid import httpexceptions as exc
+
+from h import models
+from h import paginator
+
+
+@view_config(route_name='admin_cohorts',
+             request_method='GET',
+             renderer='h:templates/admin/cohorts.html.jinja2',
+             permission='admin_cohorts')
+@paginator.paginate
+def cohorts_index(context, request):
+    return models.FeatureCohort.query.order_by(models.FeatureCohort.created.desc())
```

Comment:
This might not be too important for the admin views, but in general it's a good idea to scope all model queries through the same transaction as the request is in with:
```python
return request.db.query(models.FeatureCohort).order_by(models.FeatureCohort.created.desc())
```